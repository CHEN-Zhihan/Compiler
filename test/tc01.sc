// integer test 
geti(a);
b = a;
c = 99999;
puti_(b);
puti(c);
