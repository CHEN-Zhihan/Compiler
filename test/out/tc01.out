599999

---
699999

---
1099999

